### PR TITLE
Unicode literals

### PR DESCRIPTION
**Pull request by Reaver01**
### Changes:
- Use unicode literals instead of checking system type

@OpenPoGo/maintainers

### CODE REVIEW

**Comment by kowloon** [comment]
I got this error, seem like revert the order can fix it
File "/user/OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py", line 2
    from **future** import unicode_literals
SyntaxError: from **future** imports must occur at the beginning of the file

**Comment by Reaver01** [comment]
@kowloon - I moved it.  I didn't see that until now.

**Comment by 1karu5** [comment]
works fine here

**Comment by Reaver01** [comment]
Will see what travis-ci says about it now.  
Should be good now.

File: plugins/spin_pokestop/__init__.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
+from __future__ import unicode_literals
 import time
+import platform
```
Comment (1karu5) [inline plugins/spin_pokestop/__init__.py]: we dont need this?

File: pokemongo_bot/navigation/fort_navigator.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
+from __future__ import unicode_literals
 from datetime import datetime
+import platform
```
Comment (1karu5) [inline pokemongo_bot/navigation/fort_navigator.py]: same here?

**Comment by Reaver01** [comment]
Good catch. I will remove that

**Comment by 1karu5** [comment]
should we insert from **future** import unicode_literals in every file?
and i think you should squash your commits ;)

**Comment by Reaver01** [comment]
@1karu5 anything that has the possibility of sending unicode to logger.  Does anything else grab names that could potentially be unicode?

**Comment by Reaver01** [comment]
Can usernames on ptc or google be unicode chars?

**Comment by Reaver01** [comment]
It will get squashed when it's merged.
Can't squash them right now anyways, I am at work working off github.com and not locally.

**Comment by GucciChen** [comment]
please help .....
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 11: ordinal not in range(128)
kali rolling .

**Comment by 1karu5** [comment]
@Reaver01 
dont know. i think they can...
but may it hurt to add it everywhere? for future log messages?

**Comment by 1karu5** [comment]
@GucciChen are you using these commits here?

**Comment by Reaver01** [comment]
I guess add it to any files that import logger

**Comment by GucciChen** [comment]
@1karu5 yes!!

**Comment by GucciChen** [comment]
@1karu5 i use  https://github.com/Reaver01/OpenPoGoBot.git and also UnicodeEncodeError

**Comment by Reaver01** [comment]
that should be all of them.  
@GucciChen are you using this branch? https://github.com/Reaver01/OpenPoGoBot/tree/unicode_literals

**Comment by GucciChen** [comment]
@Reaver01  i just git clone https://github.com/Reaver01/OpenPoGoBot.git and git pull ... I am a newcomer

**Comment by Reaver01** [comment]
you need to switch to the unicode_literals branch  
git checkout unicode_literals

**Comment by 1karu5** [comment]
@GucciChen
or wait for this to get merged

@Reaver01
looks good

**Comment by GucciChen** [comment]
@Reaver01 remote: Counting objects: 11, done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 11 (delta 3), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (11/11), done.
From https://github.com/Reaver01/OpenPoGoBot
   bd62aa9..e1811e3  unicode_literals -> origin/unicode_literals

and also Traceback (most recent call last):
  File "pokecli.py", line 347, in <module>
    main()
  File "pokecli.py", line 340, in main
    bot.run()
  File "/home/OpenPoGoBot/pokemongo_bot/**init**.py", line 92, in run
    self.navigator.navigate(map_cells)
  File "/home/OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py", line 43, in navigate
    logger.log("[#] Walking towards PokeStop \"{}\" ({} away)".format(fort_name, format_dist(dist, unit)))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 11: ordinal not in range(128)

**Comment by 1karu5** [comment]
@GucciChen 
"cat /home/OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py" in console...is there "from **future** import unicode_literals" in first line?

**Comment by Reaver01** [comment]
I don't know why it's working for everyone but you.

**Comment by GucciChen** [comment]
i am go home,thatwas my work computer,tomorrow iwill try again. thanks

File: pokemongo_bot/logger.py [hunk 3]
```diff
@@ -43,7 +44,6 @@ def _log(text="", color="black", prefix=None):
     output += string
     if color in color_hex:
         output = color_hex[color] + output + Style.RESET_ALL
-    output = output.encode('utf-8')
-    print(output.decode('utf-8'))
+    print(output)
```
Comment (epicblood) [inline pokemongo_bot/logger.py:47]: should be `print(output.encode('utf-8', 'ignore')`

Comment (1karu5) [inline pokemongo_bot/logger.py:47]: could you explain why?

**Pull request closed without merging**